app/views/errorMessage.scala.html

```diff
@@ -0,0 +1,5 @@
+@import views.html.helper._
+@(elements: FieldElements)
+@if(elements.hasErrors) {
+    <span id="error" class="govuk-error-message">@elements.errors.mkString(", ")</span>
```

Comment:
Just in case it wasn't an intentional change - I think the previous version of this had "Error: " before the list of errors.

Reply:
I've put it back